lib/sentry/integrations/oban/cron.ex

```diff
@@ -8,23 +8,23 @@ defmodule Sentry.Integrations.Oban.Cron do
     [:oban, :job, :exception]
   ]
 
-  @spec attach_telemetry_handler() :: :ok
-  def attach_telemetry_handler do
-    _ = :telemetry.attach_many(__MODULE__, @events, &__MODULE__.handle_event/4, :no_config)
+  @spec attach_telemetry_handler(keyword()) :: :ok
+  def attach_telemetry_handler(config \\ []) do
```

Comment:
When is this ever called as just `attach_telemetry_handler()`? I don't think it is and if we add opts I don't think it _should_, so let's change this to


```suggestion
  def attach_telemetry_handler(config) when is_list(config) do
```